pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -181,9 +181,11 @@
      * Set the timeout for unacked messages, truncated to the nearest millisecond. The timeout needs to be greater than
      * 1 second.
      *
-     * <p>By default, the acknowledge timeout is disabled and that means that messages delivered to a
-     * consumer is not re-delivered unless the consumer crashes. Since 2.3.0, when a dead letter policy
-     * is specified, and no ackTimeoutMillis is specified, the ack timeout is set to 30 seconds.
+     * <p>By default, the acknowledge timeout is disabled (set to ‘0’ which means no limit). 
+     * When a consumer with an infinite acknowledgment timeout terminates, any unacknowledged 
+     * messages that it receives are re-delivered to another consumer.
+     * Since 2.3.0, when a dead letter policy is specified and no ackTimeoutMillis is specified, 
+     * the ack timeout is set to 30 seconds.
```

Comment:
Have you run checks to test line 184-188? Can we use ’’ in code description? What does the preview look like？

Reply:
Yes, it can be used. I've attached the screenshot in the PR description.